.changeset/four-badgers-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@utilitywarehouse/web-ui': patch
```

Comment:
same comment as [this](https://github.com/utilitywarehouse/design-systems/pull/169#discussion_r1311556524)